src/external/VTK.jl

```diff
@@ -0,0 +1,79 @@
+# Uses WriteVTK.jl to convert various structures and matrices to VTk file format
+using WriteVTK
+
+"""
+    EnergyToVTK(VTKFileName::String, energy::Energies{::Number})
+
+Stores the Energy structure in a VTK image file and returns the name of the file.
+"""
+function EnergyToVTK(VTKFileName::String, energy::Energies{<:Number})
+
+    vtkfile = vtk_grid(VTKFileName, 1, 1)
+    for i in keys(energy)
+        vtkfile[i] = energy.energies[i]
+    end
+    out = vtk_save(vtkfile)
+    return out[1]
+end
+
+"""
+    DensityToVtk(VTKFileName::String, ρ)
+
+Stores the energy density in a VTK image file and returns the name of the file.
+The density is stored in 3 submatrices : Real, Real components of Fourier, Imaginary
+components of Fourier.
+"""
+function DensityToVtk(VTKFileName::String, ρ::RealFourierArray{T, <: AbstractArray{T, 3}, 
+                                             <: AbstractArray{Complex{T}, 3}}) where T<:Real
+
+    out = vtk_write_array(VTKFileName,(ρ.real, Base.getproperty.(ρ.fourier, :re),
```

Comment:
In many calculations there is also a non-zero `ρspin` that should be dumped to VTK.